app/build.gradle

```diff
@@ -5,7 +5,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.1.2'
+        classpath 'com.android.tools.build:gradle:2.1.3'
```

Comment:
Ok, I re-ran the tests on travis and they seem to be working. So this probably isn't affecting the tests. However, let's still revert it unless you need 2.1.3 for something.